### DISCUSSION TITLE
Crud Service: create collection as time series

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#306] [Q&A] [answered]
### Which section?

Fast Data

### Which is the version?

Latest

### Description

Hi

Is it possible to create [a time series collection](https://www.mongodb.com/docs/manual/core/timeseries-collections/) with the crud service?

**Comment by fredmaggiowski** [comment, marked as answer]
Hey, Time Series collections require some additional configurations that are not covered within the CRUD Service configuration file; therefore I don't think they are supported at the moment.

Regardless this seems like a nice feature to have, I'd [open an issue](https://github.com/mia-platform/crud-service/issues/new?assignees=&labels=enhancement&projects=&template=feature-request.md&title=) on the [CRUD service repo](https://github.com/mia-platform/crud-service)! :)


@danibix95 FYI

**Reply by flower-of-the-bridges** [reply]
Hi @fredmaggiowski thanks! Here's [the issue](https://github.com/mia-platform/crud-service/issues/175)
